android/app/src/main/java/org/openbot/CameraActivity.java

```diff
@@ -240,11 +239,7 @@ protected void onCreate(final Bundle savedInstanceState) {
         new ViewTreeObserver.OnGlobalLayoutListener() {
           @Override
           public void onGlobalLayout() {
-            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
What about old Android versions?

Reply:
App's minimum supported version is 21. This check was checking for 19.

Reply:
Yes, makes sense. Android 5.0 (API level 21) = Lollipop.